rd_ui/app/scripts/visualizations/cohort.js

```diff
@@ -2,6 +2,13 @@
     var cohortVisualization = angular.module('redash.visualization');
 
     cohortVisualization.config(['VisualizationProvider', function(VisualizationProvider) {
+
+      var editTemplate = '<cohort-editor></cohort-editor>';
+        var defaultOptions = {
+          timeInterval: 'daily',
+          timeLabel: 'day'
+        };
+
         VisualizationProvider.registerVisualization({
```

Comment:
You need to pass the editTemplate to the `VisualizationProvider.registerVisualization` call as `editorTemplate` value.